doc/services/llext/config.rst

```diff
@@ -0,0 +1,102 @@
+Configuration
+#############
+
+The following Kconfig options are available for the LLEXT subsystem:
+
+.. _llext_kconf_heap:
+
+Heap size
+----------
+
+The LLEXT subsystem requires a heap to be allocated for storing extension
+related data. The size of this heap can be configured using the Kconfig option
+:kconfig:option:`CONFIG_LLEXT_HEAP_SIZE`.
```

Comment:
maybe make this a "term" just like the ones you have when enumerating the ELF object types in the following section. This way, all Kconfigs will stand out as "top-level"